lib/matplotlib/tests/test_axes.py

```diff
@@ -2341,6 +2341,16 @@ def test_manage_xticks():
     assert_array_equal(old_xlim, new_xlim)
 
 
+@cleanup
+def test_size0_ticks():
```

Comment:
can you name this `test_tick_space_size_0` so it refers to the function being tested?